orders/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls  import path
+from .views       import OrderView
+
+urlpatterns = [
+    path('/item/<int:item_id>', OrderView.as_view()),
+    path('', OrderView.as_view()),
```

Comment:
```suggestion
    path('/<int:item_id>', OrderView.as_view()),
    path('', OrderView.as_view()),
```
구매할 수 있는 상품이 item 외에 더 있는 게 아니라면 생략해도 될 거 같습니다.